schema.prisma

```diff
@@ -29,6 +29,7 @@ model Setting {
   channel       String?
   finishedSetup Boolean  @default(false)
   playlistLimit Int      @default(50)
+  announceSongs Boolean  @default(true)
```

Comment:
```suggestion
  announceSongs Boolean  @default(false)
```

I think this should probably default to false so we don't change existing behavior.